code-challenges/map-method/challenges-07.test.js

```diff
@@ -0,0 +1,252 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1
+
+Write a function named forLoopTwoToThe that, given an array of integers as input, iterates over the array and returns a new array. The returned array should contain the result of raising 2 to the power of the original input element.
+
+You may choose to complete this challenge using a for loop, for...in syntax, or for...of syntax.
+
+For example, twoToThe([1,2,3]) returns [2,4,8] because 2 ^ 1 = 2, 2 ^ 2 = 4, and 2 ^ 3 = 8.
+------------------------------------------------------------------------------------------------ */
+
+const forLoopTwoToThe = (arr) => {
+  // Solution code here...
+  let powerArray = arr.map ((value,idx) =>
```

Comment:
This was supposed to be a for loop, not a map. 

Map comes in a later challenge.

Reply:
sorry, but I was stuck with time but I solved it today by using for loop :)